Dockerfile

```diff
@@ -3,20 +3,49 @@ FROM ${BUILDPLATFORM}alpine:3
 
 ARG KUBE_VERSION
 ARG HELM_VERSION
+ARG SOPS_VERSION
+ENV HELM_SECRETS_VERSION=3.12.0
+ENV GLIBC_VERSION=2.31-r0
 ARG TARGETOS
 ARG TARGETARCH
 
 RUN apk -U upgrade \
-    && apk add --no-cache ca-certificates bash git openssh curl gettext jq \
-    && wget -q https://storage.googleapis.com/kubernetes-release/release/v${KUBE_VERSION}/bin/${TARGETOS}/${TARGETARCH}/kubectl -O /usr/local/bin/kubectl \
+    && apk add --no-cache ca-certificates bash git openssh curl gettext jq binutils \
+    && wget -q https://dl.k8s.io/release/v${KUBE_VERSION}/bin/${TARGETOS}/${TARGETARCH}/kubectl -O /usr/local/bin/kubectl \
     && wget -q https://get.helm.sh/helm-v${HELM_VERSION}-${TARGETOS}-${TARGETARCH}.tar.gz -O - | tar -xzO ${TARGETOS}-${TARGETARCH}/helm > /usr/local/bin/helm \
-    && chmod +x /usr/local/bin/helm /usr/local/bin/kubectl \
+    && curl -qsL https://github.com/mozilla/sops/releases/download/v${SOPS_VERSION}/sops-v${SOPS_VERSION}.${TARGETOS} -o /usr/local/bin/sops \
+    && chmod +x /usr/local/bin/helm /usr/local/bin/kubectl /usr/local/bin/sops \
     && mkdir /config \
-    && chmod g+rwx /config /root \
+    && chmod g+rwx /config /root /usr/local/bin/sops \
     && helm repo add "stable" "https://charts.helm.sh/stable" --force-update \
+    && helm plugin install https://github.com/jkroepke/helm-secrets --version v${HELM_SECRETS_VERSION} \
     && kubectl version --client \
-    && helm version
-
+    && helm version\
+    && curl -sL https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub -o /etc/apk/keys/sgerrand.rsa.pub \
```

Comment:
Why not just use available package for alpine to install aws-cli?
https://stackoverflow.com/a/71369083